From 5305d610c7d0855d1b9e5a3d9b2f4a97c14cd714 Mon Sep 17 00:00:00 2001
From: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Sun, 11 Oct 2015 10:12:16 -0700
Subject: Use a local copy of object.inv from doc.python.org, instead of
 downloading it each time from the internet

Patch-Name: 10_use_local_python.org_object.inv_sphinx.diff
---
 doc/source/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 3324982..6ddeafe 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -199,7 +199,7 @@ texinfo_documents = [
 # -----------------------------------------------------------------------------
 # Intersphinx configuration
 # -----------------------------------------------------------------------------
-intersphinx_mapping = {'http://docs.python.org/dev': None}
+intersphinx_mapping = {'http://docs.python.org/dev': '../../debian/python.org_objects.inv'}
 
 
 # -----------------------------------------------------------------------------
